portfolio/src/main/webapp/script.js

```diff
@@ -17,12 +17,59 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['I\'m ready, I\'m ready, I\'m ready...', 'You know what\'s funnier than 24? 25!', 'East?! I thought you said Weest!', 'Squidwaard, I used your clarinet to unclog my toilet!'];
+      ['I\'m ready, I\'m ready, I\'m ready...', 
+      'You know what\'s funnier than 24? 25!', 
+      'East?! I thought you said Weest!', 
```

Comment:
```suggestion
      'East?! I thought you said West!', 
```

Reply:
I updated it to say Weast instead because in the show, it is pronounced incorrectly as part of the joke.

Done

Reply:
you can keep it as Weast, but it was typed as Weest.